analyzer/codechecker_analyzer/analyzers/clangtidy/output_converter.py

```diff
@@ -280,33 +280,38 @@ def _get_checker_category(checker):
             return parts[0]
 
     @staticmethod
-    def _create_diag(message, fmap, files):
+    def _create_diags(message, fmap, files):
         """
         Creates a new plist diagnostic from a single clang-tidy message.
```

Comment:
```suggestion
        Creates new plist diagnostics from a single clang-tidy message.
        In case of checker aliases (i.e. `message.checker` contains multiple
        checker names separated by `,` character) the returning list contains
        a diagnostic section for each alias.
```

Reply:
At least the commit message contains the reasons of the change and describes how it is possible to get multiple diagnostics from one message, so ok.